IS2lab1/src/is2lab1/IS2lab1.java

```diff
@@ -43,39 +42,11 @@ public static void main(String[] args) {
                     menu = reader.nextInt();
                     break;
                 case 1:  
-                    User s = new User();
-                    System.out.print("\nNuevo Usuario");
-                    System.out.print("\nNombre: ");
-                    s.setUserName(sc.nextLine());
-                    System.out.print("\nEmail: ");
-                    s.setUserEmail(sc.nextLine());
-                    userCounter++;
-                    s.setUserId(userCounter);
-                    users.add(s);
+                    hp.createObject(objects, users, objectCounter);

```

Comment:
Twice createObject, did you mean to put createUser?

Reply:
yeah you are right I will fix it